corehq/apps/app_manager/models.py

```diff
@@ -4824,6 +4825,12 @@ def releases_list_json(self, timezone):
             'jar_path': self.get_jar_path(),
             'short_name': self.short_name,
             'enable_offline_install': self.enable_offline_install,
+            'include_media': self.doc_type != 'RemoteApp',
+            'j2me_enabled': 'menu_item_label' in CommCareBuildConfig.j2me_enabled_config_labels(),
```

Comment:
Did this change?  Here we check for ```'menu_item_label'``` instead of ```copy['menu_item_label']```

Reply:
oops, good catch: d8436f059fe6b43204da5b1adda791d996d3e5be